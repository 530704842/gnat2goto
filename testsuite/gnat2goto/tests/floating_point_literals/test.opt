ALL XFAIL CBMC reads wrong value of the non-zero fraction
