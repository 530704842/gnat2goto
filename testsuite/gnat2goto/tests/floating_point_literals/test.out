[1] : FAILURE
[3] : FAILURE
[2] : SUCCESS
[4] : SUCCESS
VERIFICATION FAILED
